lib/taurus/qt/qtgui/table/taurusvaluestable.py

```diff
@@ -384,6 +384,105 @@ def getReadValue(self, index):
         return self._rtabledata[index.row(), index.column()]
 
 
+class TaurusValuesSlicedArrayModel(TaurusValuesIOTableModel):
+    '''
+    Table model that allows representing 1D data in 2D format with optionally
+    different number of items per row
+    '''
+
+    def __init__(self, slices=[], parent=None):
+        self._parent = parent
+        name = self.__class__.__name__
+        TaurusValuesIOTableModel.__init__(self, [1, 1], parent)
+        self.slices = slices
+        self._super = super(TaurusValuesSlicedArrayModel, self)
```

Comment:
I think that it is better not to keep this hard reference to super. Better use super explicitly, or simply use `TaurusValuesIOTableModel`